src/tox_lsr/test_scripts/runcollection.sh

```diff
@@ -51,13 +57,20 @@ line_length_warning tox.ini
 
 # unit testing not working yet - will need these and more
 #export RUN_PYTEST_UNIT_DIR="$role/unit"
-#export PYTHONPATH="$LSR_TOX_ENV_DIR/ansible_collections/"${LSR_ROLE2COLL_NAME}"/"${LSR_ROLE2COLL_NAME}"/plugins/modules:$LSR_TOX_ENV_DIR/ansible_collections/"${LSR_ROLE2COLL_NAME}"/"${LSR_ROLE2COLL_NAME}"/plugins/module_utils"
-tox -e "$testlist" 2>&1 | tee "$LSR_TOX_ENV_DIR"/collection.tox.out || :
+#export PYTHONPATH="$MY_LSR_TOX_ENV_DIR/ansible_collections/"${LSR_ROLE2COLL_NAME}"/"${LSR_ROLE2COLL_NAME}"/plugins/modules:$MY_LSR_TOX_ENV_DIR/ansible_collections/"${LSR_ROLE2COLL_NAME}"/"${LSR_ROLE2COLL_NAME}"/plugins/module_utils"
+tox -e "$testlist" 2>&1 | tee "$MY_LSR_TOX_ENV_DIR"/collection.tox.out || :
+
+rval_adoc=0
+rval_atest=0
+if [ "${LSR_ROLE2COLL_RUN_ANSIBLE_TESTS:-}" = "true" ]; then
+    ${SCRIPTDIR}/runansible-doc.sh; rval_adoc=$? || :
+    ${SCRIPTDIR}/runansible-test.sh; rval_atest=$? || :
```

Comment:
The `|| :` here are in the wrong place - they need to be immediately after the command that returns the error - so
```bash
    ${SCRIPTDIR}/runansible-doc.sh || rval_adoc=$?
    ${SCRIPTDIR}/runansible-test.sh || rval_atest=$?
```